README.md

```diff
@@ -3,13 +3,13 @@
 Icinga DB is a set of components for publishing, synchronizing and
 visualizing monitoring data in the Icinga ecosystem, consisting of:
 
-* The Icinga DB daemon, which synchronizes monitoring data between a Redis server and a database
+* The Icinga DB daemon, which synchronizes monitoring data between a Redis®[*](doc/TRADEMARKS.md#redis) server and a database
```

Comment:
The asterisk should maybe be escaped as it has a meaning in Markdown.

```suggestion
* The Icinga DB daemon, which synchronizes monitoring data between a Redis®[\*](doc/TRADEMARKS.md#redis) server and a database
```

Reply:
I guess, it depends on the parser and linter as there are millions of Markdown dialects out there. At least my editor complained about the single asterisk.